internal/graph/dotgraph.go

```diff
@@ -481,7 +475,11 @@ func (b *builder) tagGroupLabel(g []*Tag) (label string, flat, cum int64) {
 	if dc != 0 {
 		c = c / dc
 	}
-	return formatTag(min.Value, min.Unit) + ".." + formatTag(max.Value, max.Unit), f, c
+
+	// tags are not scaled with the selected output unit because tags are often much smaller
```

Comment:
Nits:

* Capitalize the first word ("Tags") so that the first sentence is a proper one.
* It's common to format the comment text in 80 columns.